tools/targets/__init__.py

```diff
@@ -386,12 +386,7 @@ class TEENSY3_1Code(object):
     @staticmethod
     def binary_hook(t_self, resources, elf, binf):
         """Hook that is run after elf is generated"""
-        from intelhex import IntelHex
-        binh = IntelHex()
-        binh.loadbin(binf, offset=0)
-
-        with open(binf.replace(".bin", ".hex"), "w") as file_desc:
-            binh.tofile(file_desc, format='hex')
+        pass
```

Comment:
@theotherjimmy At this point can we just remove this function? Or would you prefer to do this in another PR?